src/main/kotlin/io/snyk/plugin/ui/SnykSettingsDialog.kt

```diff
@@ -178,7 +178,13 @@ class SnykSettingsDialog(
 
             cliPathTextBoxWithFileBrowser.text = applicationSettings.cliPath
             cliBaseDownloadUrlTextField.text = applicationSettings.cliBaseDownloadURL
-            additionalParametersTextField.text = applicationSettings.getAdditionalParameters(project)
+            service<FolderConfigSettings>().getAllForProject(project)
+            additionalParametersTextField.text = project.basePath?.let {
+                service<FolderConfigSettings>().getAdditionalParams(
+                    it
+                )
+            }
```

Comment:
I'm not sure I understand the logic here. `project.basePath` should not be used if it can be avoided. Why not iterate over the return values of `getAllForProject` and apply the first we find? In the future, we'll need to work on content roots, not projects, as configuration can differ by content root.